api/src/main/java/org/pmiops/workbench/api/BugReportController.java

```diff
@@ -36,7 +36,7 @@
       msg.setFrom(new InternetAddress("all-of-us-workbench-eng@googlegroups.com"));
       msg.addRecipient(Message.RecipientType.TO, new InternetAddress("all-of-us-workbench-eng@googlegroups.com", "AofU Workbench Engineers"));
       msg.setSubject("[AofU Bug Report]: " + bugReport.getShortDescription());
-      msg.setText(bugReport.getReproSteps());
+      msg.setText(bugReport.getReproSteps() + "\n\n\nTo contact the reporter of this bug, email:\n" + bugReport.getContactEmail());
```

Comment:
Does `setText` escape HTML entities, or do we need to be doing that? (What happens if you submit a bug report with `<script>alert('hi')</script>` or `\nFrom: Brian R <brew@broadinstitute.org>\n`?)

Reply:
I'll test that. Should be fine, but good thing to check.

Reply:
It does handle escaping the text. (Typing `<script>alert('hello')</script>` appeared as such in the email. Similarly with: `/nhello/n/nbut/n/n/nnewlines?`)